data/json/professions.json

```diff
@@ -2667,6 +2667,193 @@
+                "bra",
+                "panties"
+            ]
+        }
+    },
+	{
+        "type": "profession",
+        "ident": "imam",
+        "name": {
+            "male": "Imam",
+            "female": "Mourchida"
+        },
+        "description": "You spent much of your time prior to the apocalypse at the local mosque, studying the words of the prophet and the Quran, and guiding your community in prayer. Back then they came from far and wide to listen to you, now they come to eat your brains.",
+        "points": 0,
+		"skills": [
+            {
+                "level": 2,
+                "name": "speech"
+            },
+            {
+                "level": 1,
+                "name": "barter"
+            }
+        ],
+        "items": {
+            "both": [
+                "pants",
+                "tshirt",
+                "socks",
+                "kufi",
+                "thawb",
+                "lowtops",
+                "holybook_quran"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "bra",
+                "panties"
+            ]
+        }
+    },
+	{
+        "type": "profession",
+        "ident": "rabbi",
+        "name": "Rabbi",
+        "description": "You were celebrating with your flock in the temple when the apocalypse struck. You sure could use a messiah right now!",
+        "points": 0,
+		"skills": [
+            {
+                "level": 2,
+                "name": "speech"
+            },
+            {
+                "level": 1,
+                "name": "firstaid"
+            }
+        ],
+        "items": {
+            "both": [
+                "pants",
+                "leather_belt",
+                "dress_shirt",
+                "socks",
+                "kittel",
+                "kippah",
+                "dress_shoes",
+                "holy_symbol",
+                "holybook_talmud",
+                "holybook_tanakh"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "bra",
+                "panties"
+            ]
+        }
+    },
+	{
+        "type": "profession",
+        "ident": "guru",
+        "name": "Guru",
+        "description": "You spent many years traveling through the world, becoming wise and learned. Normally, you can answer any question, but even you are not quite sure what to do about the ravenous undead.",
+        "points": 1,
+		"skills": [
+            {
+                "level": 2,
+                "name": "speech"
+            },
+            {
+                "level": 1,
+                "name": "survival"
+            }
+        ],
+        "items": {
+            "both": [
+                "jeans",
+                "tshirt",
+                "socks",
+                "robe",
+                "turban",
+                "waterskin",
+                "pockknife",
+                "bag_canvas",
```

Comment:
Why would a guru haul a [canvas sack](http://thumbs.dreamstime.com/x/burlap-sacks-potatoes-8293765.jpg) around? It's not an ordinary bag made out of canvas, you'd need a messenger bag (mbag) for closest approximation of flap-and-strap bag.